locust/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# define the liboqs tag to be used
+ARG LIBOQS_TAG=main
+
+# define the oqsprovider tag to be used
+ARG OQSPROVIDER_TAG=main
+
+# define the Curl version to be baked in
+ARG OPENSSL_BRANCH="master"
+
+# Default location where all binaries wind up:
+ARG INSTALLDIR=/opt/oqssa
+
+# Default Python version to be used
+ARG PYTHON_VERSION=3.12.6
+
+# liboqs build type variant; maximum portability of image:
+ARG LIBOQS_BUILD_DEFINES="-DOQS_DIST_BUILD=ON"
+
+# Default root CA signature algorithm; can be set to any listed at https://github.com/open-quantum-safe/oqs-provider#algorithms
+ARG SIG_ALG="dilithium3"
+
+# Default KEM algorithms; can be set to any listed at https://github.com/open-quantum-safe/oqs-provider#algorithms
+ARG DEFAULT_GROUPS="x25519:x448:kyber512:p256_kyber512:kyber768:p384_kyber768:kyber1024:p521_kyber1024"
+
+# Define the degree of parallelism when building the image; leave the number away only if you know what you are doing
+ARG MAKE_DEFINES="-j 16"
+
+FROM alpine:3.20.3
+# Take in all global args
+ARG LIBOQS_TAG
+ARG OQSPROVIDER_TAG
+ARG CURL_VERSION
```

Comment:
Looks like this arg is never used—I'm assuming it's a copy-paste holdover from the curl dockerfile?

Reply:
Yes it was a mistake during the first copy/paste, many thanks for all comments!!. 
Fix in this [commit](https://github.com/open-quantum-safe/oqs-demos/pull/304/commits/91769ca63c25a4b9fb393634cf6ae2397f87d132)